assets/web/config.html

```diff
@@ -629,6 +625,35 @@ <h1 class="my-4">Configuration</h1>
         </div>
       </div>
     </div>
+    <!-- Intel QuickSync Settings-->
+    <div v-if="currentTab === 'qsv'" class="config-page">
+      <div class="mb-3">
+        <label for="qsv_preset" class="form-label">QuickSync Presets</label>
+        <div class="form-text">
+          <label for="qsv_preset" class="form-label">QuickSync Preset</label>
+          <select id="qsv_preset" class="form-select" v-model="config.qsv_preset">
+            <option value="7">Very Fast</option>
+            <option value="6">Faster</option>
+            <option value="5">Fast</option>
+            <option value="4">Medium</option>
+            <option value="3">Slow</option>
+            <option value="2">Slower</option>
+            <option value="1">Very Slow</option>
+          </select>
+        </div>
+      </div>
```

Comment:
@TheElixZammuto seems for every encoder we add a new setting for preset... can we have a single setting for preset that applies to whatever encoder is selected? I believe the possible values are the same for all of them.

Reply:
Unfortunately this is not the case since x264,amf, nvenc and qsv all have different preset options